web/src/app/chat/modal/configuration/LlmTab.tsx

```diff
@@ -84,9 +73,12 @@ export function LlmTab({
             llmOverride.provider,
             llmOverride.modelName
           )}
-          onSelect={(value) =>
-            setLlmOverride(destructureValue(value as string))
-          }
+          onSelect={(value) => {
+            if (chatSessionId) {
```

Comment:
does this result in a bit of slowness when switching models? Should we put this after `setLlmOverride`?